rules/falco_rules.yaml

```diff
@@ -984,8 +984,8 @@
     rule "Read sensitive file untrusted".
   condition: >
     create_hardlink 
-    and (evt.arg.oldpath in (sensitive_file_names))
-  output: Hardlinks created over sensitive files (target=%evt.arg.target linkpath=%evt.arg.linkpath evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
+    and (fs.path.source in (sensitive_file_names))
+  output: Hardlinks created over sensitive files (target=%fs.path.target linkpath=%fs.path.source evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty %container.info)
```

Comment:
`evt.arg.target` and `evt.arg.linkpath` don't exist for `link` and `linkat`

Reply:
CC @darryk10 and @loresuso to also confirm, thanks!

Reply:
Actually this was a refactor mistake (by me since I had to do it all), I would instead suggest to revert to the old correct outputs: `target=%evt.arg.oldpath linkpath=%evt.arg.newpath`, see in the git history https://github.com/falcosecurity/rules/blob/dc90ca5e179fcf11e6a9b8486e4f23dacb613087/rules/falco_rules.yaml#L2808C126-L2808C175

Reply:
Because using the new `fs.path*` fields would break things and it would be inconsistent with the symlink rule.

Reply:
done, thank you for the feedback!